LiteCore/Support/SecureRandomize.hh

```diff
@@ -38,6 +38,51 @@
     }
 
     #define SECURE_RANDOMIZE_AVAILABLE 1
+    
+#elif defined(_CRYPTO_MBEDTLS)
+
+    #include <mbedtls/entropy.h>
+    #include <mbedtls/ctr_drbg.h>
+
+#ifdef MBEDTLS_NO_PLATFORM_ENTROPY
+    #include <Windows.h>
```

Comment:
This seems to assume that if `MBEDTLS_NO_PLATFORM_ENTROPY` is defined, we must be on Windows. Is that a safe assumption to make?